core/src/main/java/services/servicesImpl/POISeviceImpl.java

```diff
@@ -0,0 +1,48 @@
+package services.servicesImpl;
+
+import entities.HeatMap;
+import entities.HeatPoint;
+import entities.POI;
+import repository.POIRepository;
+import services.POIService;
+import services.POISourceService;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class POISeviceImpl implements POIService{
+
+    public static final double cityX1 = 23.7;
```

Comment:
this can be moved to some setting file (or even to database as new entity Location)